Dockerfile

```diff
@@ -1,25 +1,46 @@
 FROM mhart/alpine-node:10 as builder
 
-RUN apk add --no-cache make gcc g++ python git bash
-COPY package.json /app/package.json
-COPY package-lock.json /app/package-lock.json
+RUN apk --update --no-cache add \
+    make \
+    gcc \
+    g++ \
+    python \
+    git \
+    bash \
+    cmake \
+    libressl-dev \
+    gmp-dev \
+    procps-dev \
+    pkgconfig \
```

Comment:
I think you can remove `pkgconfig`, `build-base`, `boost-dev` and `libxslt-dev`.

Reply:
It seems I can indeed. Thanks!